docs/sources/tempo/configuration/grafana-alloy/tail-sampling/policies-strategies.md

```diff
@@ -0,0 +1,585 @@
+| [rate_limiting](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/otelcol/otelcol.processor.tail_sampling/#rate_limiting-block)         | Samples based on rate of spans per second.                                                                     | Controlling data volume.                                                |
+| [span_count](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/otelcol/otelcol.processor.tail_sampling/#span_count-block)               | Samples based on the minimum number of spans within the observed trace.                                        | Limiting sampled data to a specific number of spans within a trace.     |
+| [status_code](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/otelcol/otelcol.processor.tail_sampling/#status_code-block)             | Samples based upon the status code, either OK, Error, or Unset.                                                | Capturing erroring traces.                                              |
+| [string_attribute](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/otelcol/otelcol.processor.tail_sampling/#string_attribute-block)   | Samples based on string attributes (resource and record) value matches.                                        | Filtering specific services or database queries.                        |
+| [trace_state](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/otelcol/otelcol.processor.tail_sampling/#trace_state-block)             | Samples based on TraceState value matches. | Implementing complex sampling strategies that rely on trace context. |
+
+
+## Sampling policies and use cases
+
+This section provides examples for the policy strategies.
+
+{{< admonition type = "note" >}}
+Sample data at the collector after metrics generation so that all traces are available to generate accurate metrics. If you generate metrics from sampled traces, the sampling affects their values.
+{{< /admonition >}}
+
+### Always sample
+
+You can use `always_sample` when you want to capture all tracing data. This could be useful for troubleshooting.
+
+Refer to the <code>[always_sample](https://grafana.com/docs/<ALLOY_VERSION>/reference/components/otelcol/otelcol.processor.tail_sampling/#always_sample)</code> documentation for more information.` `
+
+```alloy
+policy {
+    // The example below is for the `always_sample` policy
+    name = "example_always_sample"
+    type = "always_sample"
+}
+```
+
+### And
+
+You use the `and` sampling policy when you want to match on multiple conditions. This example uses a probabilistic sampler and the latency sampling policy.
+
+You can use this to look for slow requests and sample a percentage of traces.
+
+Refer to the [`and`](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/otelcol/otelcol.processor.tail_sampling/#and) documentation for more information.
```

Comment:
It seems to make more sense for all these refer to links to include the word documentation in the linked part vs just the policy name.

Reply:
I'll make this change across the file for consistency.